drivers/boston_project_scorecard/app/models/boston_project_scorecard/total_score.rb

```diff
@@ -63,6 +64,7 @@ def financial_performance_score
           invoicing_accuracy_score,
           efficiency_score,
           required_match_score,
+          supportive_services_score,
```

Comment:
should `supportive_services_score` be part of `financial_performance_score`? Maybe it should be in `financial_performance_score` instead